lib/liquid/tags/comment.rb

```diff
@@ -10,6 +10,12 @@ def unknown_tag(_tag, _markup, _tokens)
     def blank?
       true
     end
+
+    def parse(tokens)
+      @body = CommentBody.new
```

Comment:
Another approach would be to do the same as `Liquid::Raw#parse`

https://github.com/Shopify/liquid/blob/27c91203ab4adfde2cc5e1142bce2a267634ef8e/lib/liquid/tags/raw.rb#L12-L23

but it wouldn't support things like `{% comment %}{% comment %}{% endcomment %}{% endcomment %}`

```
~/Development/liquid master
❯ irb -I ./lib
irb(main):001:0> require 'liquid'
=> true
irb(main):002:0> Liquid::Template.parse('{% raw %}{% raw %}{% endraw %}{% endraw %}')
Liquid::SyntaxError: Liquid syntax error: Unknown tag 'endraw'
	from /Users/david/Development/liquid/lib/liquid/document.rb:23:in `unknown_tag'
	from /Users/david/Development/liquid/lib/liquid/document.rb:11:in `block in parse'
	from /Users/david/Development/liquid/lib/liquid/block_body.rb:33:in `parse'
	from /Users/david/Development/liquid/lib/liquid/document.rb:10:in `parse'
	from /Users/david/Development/liquid/lib/liquid/document.rb:5:in `parse'
	from /Users/david/Development/liquid/lib/liquid/template.rb:132:in `parse'
	from /Users/david/Development/liquid/lib/liquid/template.rb:116:in `parse'
	from (irb):2
	from /usr/local/bin/irb:11:in `<main>'
```